### ISSUE TITLE
`containers-0.6`+?

### ISSUE THREAD

**Issue by aryairani** [roelvandijk/numerals#29]
Could you release a version that supports `containers-0.6`? The current version of containers is 0.7, but `numerals` wants `<6`.

**Comment by roelvandijk** [comment]
Good point. I'll look into it when I have some free time. A pull request is also welcome of course :-)

**Comment by aryairani** [comment]
Thanks!

Re. a PR, I considered it briefly but didn't know how, sorry! I'm more familiar with .hs than the package-related bits unfortunately.

I ended up with `allow-newer-deps: [numerals]` to tide me over in the mean time. 😅 

For what it's worth, that showed me that numerals builds and works fine for me with extended dependency bounds, so hopefully it's an easy patch.

**Comment by aryairani** [comment]
This was reported to me as well, re. the PR in which I'd used `allow-newer-deps: [numerals]`. I'll invite them to submit a PR here, but I wanted to put it on your radar, just in case.

> numerals [requires](https://hackage.haskell.org/package/numerals-0.4.1/dependencies) `text (>=0.11 && <1.3)` but LTS 22.26 has [`text 2.0.2`]

**Comment by neduard** [comment]
Ace, thanks both!  I've raised a PR.  Let me know your thoughts @roelvandijk 

edit: sorry!  I forgot to mention: turns out that the dependency requirements were already relaxed in https://github.com/roelvandijk/numerals/commit/f7ebf5f905f4a14ed65264fdfaea9fb2f632e5c6 , just that no release happened since then. So the only thing I had to do was to bump the version number.
